terraform/variables.tf

```diff
@@ -274,3 +274,11 @@ variable "website_feature_flags" {
     error_message = "Value must be JSON-serializable."
   }
 }
+
+// Passage Authentication (docs.passage.id)
+variable "passage_app_id" {
+  description = "Passage Application ID"
+  type        = string
+  default     = ""
+  sensitive   = true
```

Comment:
@TylerHendrickson I needed these to be sensitive and change based on environment. I assume we have to configure this manually in the AWS console, if not lmk how I can do that.